index.html

```diff
@@ -22,6 +22,12 @@ <h1> Search repos </h1>
             <button class="submit-button" type="submit"> Submit </button>
         </form>
         <div id="dinamicContent"> </div>
+        <ul class="repo-list">
+            <li class="skeleton-card m-3"> </li>
+            <li class="skeleton-card m-3"> </li>
```

Comment:
The cards should have the `repo-card` class. This way, they have the exact same values as the repo cards.

However, we want some extra stuff, so you can have 2 classes here:

```
repo-card repo-card--skeleton
```